website/docusaurus.config.js

```diff
@@ -145,5 +145,8 @@ module.exports = {
       },
     ],
   ],
-  plugins: ['docusaurus-plugin-sass'],
+  plugins: [
+    'docusaurus-plugin-sass',
+    ['@docusaurus/plugin-client-redirects', { fromExtensions: ['html'] }],
```

Comment:
easier than I've imagined 🤣

Reply:
Partly why this framework is so great! 😎